plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/Completions.kt

```diff
@@ -51,10 +53,12 @@ internal object Completions {
 
             is FirUnknownPositionContext -> {
                 complete(factory.keywordContributor(0), positionContext)
+                complete(factory.declarationFromUnresolvedNameContributor(0), positionContext) // for parameter declaration
```

Comment:
Please, create new position context for such case.

Reply:
Thanks for the suggestion! Added `FirValueParameterPositionContext` for this case.